packages/app-config-writer/README.md

```diff
@@ -1,6 +1,12 @@
 # @sap-ux/app-config-writer
 
-The `@sap-ux/app-config-writer` adds smartlinks configuration for a SAP UX project. 
+The `@sap-ux/app-config-writer` adds or updates configurations to a SAP UX project.
```

Comment:
SAP Fiori tools project is better aligned with branding ;)

Reply:
Thanks for the suggestion, changed